### DISCUSSION TITLE
My ques. is for nikse dk

### DISCUSSION THREAD

**Discussion by Mad-Lee-19** [SubtitleEdit/subtitleedit#7877] [Q&A] [answered]
Hi there, I downloaded subtitle edit 4.0.3 recently but it won't play any video I upload on it, shows only the waveform of the video but can't play nor generate anything on it

**Comment by darnn** [comment]
Click the layout button, for me it looks like this:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/19672444/a3f81b75-4fef-41ca-9199-a9099716ba1b)

And choose a layout with a video player.

**Reply by Mad-Lee-19** [reply]
no thats not it, nothing shown on the video rectangle and can't play anything no matter which video I choose...

**Comment by darnn** [comment]
Then try going to options->settings->video player, and choosing MPV there.

**Reply by Mad-Lee-19** [reply]
Already done that too, I chose MPV and also downloaded it and still nothing...

**Comment by darnn** [comment]
In that case I'm out of ideas, but I would suggest you describe the problem in a little more detail. What happens when you try to open a video? What happens when you try to play it? And so on. Post screenshots if necessary.

**Reply by Mad-Lee-19** [reply]
When I pick a video,nothing shown on the video rectangle, its totally a black blank no even a photo for the video, and I press play nothing is played, but it shows a waveform in the waveform rectangle but I press play also nothing...

**Reply by Mad-Lee-19** [reply]
Also when I try to generate the video and then download a language for, it gives an error message, would you like me to send it to you?

**Reply by niksedk** [reply]
You can attach the error message here.

**Reply by Mad-Lee-19** [reply]
Here is the error message :

Unable to download https://alphacephei.com/vosk/models/vosk-model-en-us-0.22-lgraph.zip!

No content downloaded - missing file or no internet connection!

   at Nikse.SubtitleEdit.Forms.AudioToText.VoskModelDownload.CompleteDownload(MemoryStream downloadStream)
   at Nikse.SubtitleEdit.Forms.AudioToText.VoskModelDownload.buttonOK_Click(Object sender, EventArgs e)

**Reply by niksedk** [reply, marked as answer]
Hm, works fine here. 
Try allowing SE through your firewall... 
SE on Windows 7 can have some trouble with internet connection as TLS for dot net needs to be correctly configured.

**Reply by Mad-Lee-19** [reply]
Thanks any way...
Till I update my windows or my computer, I will get back to you,
Thank You.

**Comment by niksedk** [comment]
Sounds like a problem with your computer...

You can try:
- Re-install dot net framework 4.8 - https://dotnet.microsoft.com/en-us/download/dotnet-framework/net48 (choose Runtime)
- Re-install your video drivers
- Update Windows with latest updates
- Update Windows to latest version
